src/legacy/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -37,32 +35,76 @@ export function GaugeChartProvider(Private) {
     addEvents(element) {
       const events = this.events;
 
-      return element
-        .call(events.addHoverEvent())
-        .call(events.addMouseoutEvent());
+      return element.call(events.addHoverEvent()).call(events.addMouseoutEvent());
+    }
+
+    /**
+     * returns the displayed width and height of a single gauge depending on selected alignment
+     * @param alignment - automatic | horizontal | vertical
+     * @param containerDom
+     * @param nrOfItems
+     * @returns {{width: number, height: number}}
+     */
+    calcGaugeDim(alignment, containerDom, nrOfItems) {
+      const containerWidth = containerDom.clientWidth;
+      const containerHeight = containerDom.clientHeight;
+      const containerMargin = 20;
```

Comment:
Testing with 25 (as the bottom margin magic number) as container margin will solve some scrollbar related issues

Reply:
this was done